CONTRIBUTING.md

```diff
@@ -0,0 +1,221 @@
+
+#### 🐞 Submitting Bug Issues
+
+Resolving bugs is a priority for our project. We welcome bug reports. However, please make sure to do the following prior to submitting a bug report:
+- **Check for duplicates** - there may be a bug report already describing your issue, so check the Issue tracking system on GitHub first.
+
+Here's some guidance on submitting a bug issue:
+1. Navigate to our Issue tracker on GitHub and file a new issue
+2. Select a bug template (if available) for your issue
+   1. Fill out the template fields to the best of your ability, including output snippets or screenshots where applicable
+3. Follow the general guidelines below for extra information about your bug
+   1. Include a code snippet if you have it showcasing the bug
+   2. Provide reproducible steps of how to recreate the bug
+   3. If the bug triggers an exception or error message, include the *full message* or *stacktrace*
+   4. Provide information about your operating system and the version of our project you're using
+
+#### 🆕 Submitting New Feature Issues
+
+We welcome new feature requests to help grow our project. However, please make sure to do the following prior to submitting a new feature request:
+- **Check for duplicates** - there may be a new feature issue already describing your issue, so check the GitHub Issue Tracker first
+- **Consider alternatives** - is your feature really needed? Or is there a feature within our project or with a third-party that may help you achieve what you want?
+
+Here's some guidance on submitting a new feature issue:
+1. Navigate to our GitHub Issue Tracker and file a new issue
+2. Select a new feature template (if available) for your issue
+   1. Fill out the template fields to the best of your ability
+
+#### 🔓 Submitting Security Vulnerability Issues
+
+Security vulnerabilities should **not** be filed to the regular issue tracking system.
+
+Report your security vulnerabilities to the project administrator via their Jacobs email address.
+
+Please be sure to:
+* Indicate the severity of the vulnerability
+* Provide any workarounds, if you know them
+* Provide return-contact information to follow-up with you if needed
+
+## 🚀 Development Process
+
+This project integrates contributions from many people, and so we'd like to outline a process you can use to visualize how your contributions may be integrated if you provide something. 
+
+```mermaid
+flowchart TD
+    repo_proj[(Main Branch)]-->|Branch|branch[(Your Branch)]
+    branch-->|Make|patch(Your Changes)
+    patch-->|Submit|pr((Pull Request))
+    pr==>|Approved|repo_proj
+    pr-.->|Changes Requested|branch
+```
+
+### ✨ Make your Modifications
+
+Branching the repository is the preferred way to propose changes. 

```

Comment:
If you do not have write access to the repository, create a fork first https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo